lua/packer.lua

```diff
@@ -369,11 +369,40 @@ packer.sync = function(...)
   end)()
 end
 
+
+local function reload_module(name)
+  if name then
+    package.loaded[name] = nil
+    return require(name)
+  end
+end
+
+--- Search through all the loaded packages for those that
+--- return a function, then cross reference them with all
+--- the plugin configs and setups and if there are any matches
+--- reload the user module.
+local function precompile(plugs)
```

Comment:
Super nit-picky: I'd prefer we call this something else, like `refresh_configs` or similar. `precompile` is a bit vague.

Reply:
happy to change it 👍🏿 I just went with something vaguely applicable